subsys/mpsl/init/mpsl_init.c

```diff
@@ -388,11 +466,26 @@ static int32_t mpsl_lib_init_internal(void)
 	memset(&clock_cfg, 0, sizeof(clock_cfg));
 #endif
 
+#if !defined(CONFIG_SOC_SERIES_NRF54HX) && !defined(CONFIG_SOC_SERIES_NRF54LX)
+	/* On boot, MPSL is given some information about how much time
+	 * it takes for the high-frequency crystal to ramp up.
+	 */
+	static uint16_t hfxo_power_up_time_us;
```

Comment:
Are we sure, we do not need to re-initialize the value between CPU resets, and sufficient to be zero-ed once at startup only?